src/vivarium_public_health/disease/models.py

```diff
@@ -0,0 +1,152 @@
+
+        healthy = SusceptibleState(self.cause)
+        healthy.allow_self_transitions()
+
+        get_data_functions = {}
+        if only_morbid:
+            infected = DiseaseState(self.cause)
+            get_data_functions['csmr'] = lambda _, __: None  # DiseaseModel will try to pull not provided
+        else:
+            infected = ExcessMortalityState(self.cause) 
+        infected.allow_self_transitions()
+
+        healthy.add_transition(infected, source_data_type='rate')
+
+        builder.components.add_components([DiseaseModel(self.cause, states=[healthy, infected],
+                                                        get_data_functions=get_data_functions)])
+
+
+class SIR:
+
+    def __init__(self, cause: str):
+        self.cause = cause
+
+    def setup(self, builder):
+        only_morbid = builder.data.load(f'cause.{self.cause}.restrictions')['yld_only']
+
+        healthy = SusceptibleState(self.cause)
+        healthy.allow_self_transitions()
+
+        get_data_functions = {}
+        if only_morbid:
+            infected = DiseaseState(self.cause)
+            get_data_functions['csmr'] = lambda _, __: None  # DiseaseModel will try to pull not provided
+        else:
+            infected = ExcessMortalityState(self.cause)
+
+        infected.allow_self_transitions()
+
+        recovered = RecoveredState(self.cause)
+        recovered.allow_self_transitions()
+
+        healthy.add_transition(infected, source_data_type='rate')
+        infected.add_transition(recovered, source_data_type='rate')
+
+        builder.components.add_components([DiseaseModel(self.cause, states=[healthy, infected, recovered],
+                                                        get_data_functions=get_data_functions)])
+
+
+class SIS:
+
+    def __init__(self, cause: str):
+        self.cause = cause
+
+    def setup(self, builder):
+        only_morbid = builder.data.load(f'cause.{self.cause}.restrictions')['yld_only']
+
+        healthy = SusceptibleState(self.cause)
+        healthy.allow_self_transitions()
+
+        get_data_functions = {}
+        if only_morbid:
+            infected = DiseaseState(self.cause)
+            get_data_functions['csmr'] = lambda _, __: None  # DiseaseModel will try to pull not provided
+        else:
+            infected = ExcessMortalityState(self.cause)
+
+        infected.allow_self_transitions()
+
+        healthy.add_transition(infected, source_data_type='rate')
+        infected.add_transition(healthy, source_data_type='rate')
+
+        builder.components.add_components([DiseaseModel(self.cause, states=[healthy, infected],
+                                                        get_data_functions=get_data_functions)])
+
+
+class SIS_fixed_duration:
+
+    def __init__(self, cause: str, duration):
+        self.cause = cause
+        if not isinstance(duration, pd.Timedelta):
+            self.duration = pd.Timedelta(days=float(duration) // 1, hours=float(duration) % 1)
```

Comment:
Can we add a doc string saying how duration should be specified? This makes it seem like if I did something like duration=28.5, I'd end up with 28 days and 0.5 hours which doesn't really make sense

Reply:
@collijk can I multiply hourse by 24 ? I agree with kate, this is a little confusing

Reply:
Yep, think this is an error.